From f8a38c7e6e75c105b98c7b38148e7daedfeb5f29 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Tue, 12 Feb 2019 17:57:41 +0000
Subject: [PATCH] OS-7576 ctfdump -c should include non-root types Reviewed by:
 Robert Mustacchi <rm@joyent.com> Approved by: Robert Mustacchi
 <rm@joyent.com>

---
 usr/src/cmd/ctfdump/ctfdump.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/cmd/ctfdump/ctfdump.c b/usr/src/cmd/ctfdump/ctfdump.c
index e241c35b47..ec879fcc06 100644
--- a/usr/src/cmd/ctfdump/ctfdump.c
+++ b/usr/src/cmd/ctfdump/ctfdump.c
@@ -10,7 +10,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -982,7 +982,7 @@ ctfdump_source(void)
 		    strerror(errno));
 	}
 
-	if (ctf_type_iter(g_fp, B_FALSE, ctfsrc_collect_types_cb,
+	if (ctf_type_iter(g_fp, B_TRUE, ctfsrc_collect_types_cb,
 	    idnames) == CTF_ERR) {
 		warnx("failed to collect types: %s",
 		    ctf_errmsg(ctf_errno(g_fp)));
-- 
2.21.0

